ui/app/pages/create-account/connect-hardware/select-hardware.js

```diff
@@ -117,74 +123,95 @@ export default class SelectHardware extends Component {
     );
   }
 
-  getAffiliateLinks() {
-    const links = {
-      trezor: `<a class='hw-connect__get-hw__link' href='https://shop.trezor.io/?a=metamask' target='_blank'>Trezor</a>`,
-      ledger: `<a class='hw-connect__get-hw__link' href='https://www.ledger.com/products/ledger-nano-s?r=17c4991a03fa&tracker=MY_TRACKER' target='_blank'>Ledger</a>`,
-    };
-
-    const text = this.context.t('orderOneHere');
-    const response = text
-      .replace('Trezor', links.trezor)
-      .replace('Ledger', links.ledger);
-
-    return (
-      <div
-        className="hw-connect__get-hw__msg"
-        dangerouslySetInnerHTML={{ __html: response }}
-      />
-    );
-  }
-
-  renderTrezorAffiliateLink() {
-    return (
-      <div className="hw-connect__get-hw">
-        <p className="hw-connect__get-hw__msg">
-          {this.context.t('dontHaveAHardwareWallet')}
-        </p>
-        {this.getAffiliateLinks()}
-      </div>
-    );
+  renderTutorialsteps() {
+    switch (this.state.selectedDevice) {
+      case 'ledger':
+        return this.renderLedgerTutorialSteps();
+      case 'trezor':
+        return this.renderTrezorTutorialSteps();
+      default:
+        return '';
+    }
   }
 
-  scrollToTutorial = () => {
-    if (this.referenceNode) {
-      this.referenceNode.scrollIntoView({ behavior: 'smooth' });
+  renderLedgerTutorialSteps() {
+    const steps = [];
+    if (this.props.useLedgerLive) {
+      steps.push({
+        title: this.context.t('step1LedgerWallet'),
+        message: this.context.t('step1LedgerWalletMsg', [
+          // eslint-disable-next-line react/jsx-key
+          <a
+            className="hw-connect__msg__link"
+            href="https://www.ledger.com/ledger-live"
+            rel="noopener noreferrer"
+            target="_blank"
+          >
+            Ledger Live App
+          </a>,
+        ]),
+      });
     }
-  };
 
-  renderLearnMore() {
+    steps.push({
+      asset: 'plug-in-wallet',
+      dimensions: { width: '225px', height: '75px' },
+      title: this.context.t('step2LedgerWallet'),
+      message: this.context.t('step2LedgerWalletMsg', [
+        // eslint-disable-next-line react/jsx-key
+        <a
+          className="hw-connect__msg__link"
+          href="https://metamask.zendesk.com/hc/en-us/articles/360020394612-How-to-connect-a-Trezor-or-Ledger-Hardware-Wallet"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          {this.context.t('hardwareWalletSupportLinkConversion')}
+        </a>,
+      ]),
+    });
+
     return (
-      <p className="hw-connect__learn-more" onClick={this.scrollToTutorial}>
-        {this.context.t('learnMore')}
-        <img
-          className="hw-connect__learn-more__arrow"
-          src="images/caret-right.svg"
-          alt=""
-        />
-      </p>
+      <div
+        className="hw-tutorial"
```

Comment:
Nit: It'd be nice to avoid duplicating this part between both of the tutorial functions.

Maybe there's a better way to organize this :thinking: This whole "steps" thing doesn't seem very idiomatic. Rather than populating this array and using it to generate the steps, it might be better to return the rendered markup directly, or with the help of additional render helper methods like `renderStep`, whatever makes sense.